pkgs/tools/system/syslog-ng/default.nix

```diff
@@ -1,17 +1,39 @@
-{ stdenv, fetchurl, eventlog, pkgconfig, glib, python, systemd, perl
-, riemann_c_client, protobufc, pcre, yacc }:
+{ stdenv, fetchgit, autoconf, autoconf-archive, automake, libtool, flex, openssl
+, eventlog, pkgconfig, glib, python, systemd, perl
+, riemann_c_client, protobufc, pcre, yacc, which }:
 
-stdenv.mkDerivation rec {
-  name = "syslog-ng-${version}";
+stdenv.mkDerivation {
+  name = "syslog-ng-3.9.1";
```

Comment:
Please leave `version`. It was discussed (admittedly a long time ago and it'd be difficult to find) that we might move to using it insttead of parsing `name` some day; it'd definitely be easier to move if most packages already contain it. Disclosure: I want it to happen very much as current way seems ugly to me ^_^